src/browser/progressbar.js

```diff
@@ -18,6 +18,42 @@ define((require, exports, module) => {
   const {Element, VirtualAttribute} = require('./element');
   const Component = require('omniscient');
 
+  // Animation parameters:
+  const A = 0.2;              // Zone A size (a full progress is equal to '1'
+  const B = 0.2;              // Zone B size
+  const APivot = 200;         // When to reach ~80% of zone A
+  const BPivot = 500;         // When to reach ~80% of zone B
+  const CDuration = 200;     // Time it takes to fill zone C
+  const ApproachFunc = (tMs, pivoMs) => 2 * Math.atan(tMs / pivoMs) / Math.PI;
+
+  function ComputeProgress(viewer) {
```

Comment:
Nit: Could you please use lower leading camel case names for plain functions, and upper case leading for constructors  / factories ?